### ISSUE TITLE
Playlist with relative path

### ISSUE THREAD

**Issue by Martinxc** [vzhd1701/gridplayer#223]
### Suggestion

Would be great to make GridPlayer load videos from playlist without known absolute path. For example right now if I burn a CD with playlist, path to them will change, and won't be able to open it. But it would be great if I would save playlist in the same directory or directory above, and it would save relative paths, so I could move entire folders or burn them on a CD.

**Comment by nazarux** [comment]
> ### Suggestion
> Would be great to make GridPlayer load videos from playlist without known absolute path. For example right now if I burn a CD with playlist, path to them will change, and won't be able to open it. But it would be great if I would save playlist in the same directory or directory above, and it would save relative paths, so I could move entire folders or burn them on a CD.

I agree! ...and many other reasons - organizing your media is just another one.
